realtime/server.js

```diff
@@ -1,14 +1,16 @@
-var io = require('socket.io');
-var req = require('request');
-var fs = require('fs');
-var winston = require('winston');
+var io = require('socket.io'); //include websockets
+var req = require('request'); //include request(supports http)
+var fs = require('fs'); //include file system
+var winston = require('winston'); //include winston(multi-transport async logging library)
 
 winston.add(winston.transports.File, { filename: 'server.log' });
 winston.level = 'debug';
 
+//obtain some imporant info from common.json
```

Comment:
The fact that information is obtained from common.json is obvious from the code. Without telling what information that is, the comment is unnecessary.